Tiqr/Resources/General/Tiqr-Info.plist

```diff
@@ -52,6 +52,11 @@
 	<string>1.0</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
```

Comment:
It should use stricter settings, I discussed with @joostd to only use a whitelist here for the demo server. Other servers should be secure.